addons/actions/src/models/ActionsFunction.ts

```diff
@@ -0,0 +1,85 @@
+import { ActionOptions } from './ActionOptions';
+import { ActionsMap } from './ActionsMap';
+import { HandlerFunction } from './HandlerFunction';
+
+export interface ActionsFunction {
+  <T extends string>(handlerMap: Record<T, string>, options?: ActionOptions): ActionsMap<T>;
+  <T extends string>(...handlers: T[]): ActionsMap<T>;
+
+  <T extends string>(handler1: T, options?: ActionOptions): ActionsMap<T>;
+  <T extends string>(handler1: T, handler2: T, options?: ActionOptions): ActionsMap<T>;
+  <T extends string>(handler1: T, handler2: T, handler3: T, options?: ActionOptions): Record<
```

Comment:
It could be:
```ts
  <T extends string>(handler1: T, handler2: T, handler3: T, options?: ActionOptions): ActionsMap<T>;
```
Or am I missing something?

Reply:
Whoops! It's fixed now. Thanks for the review.